.jvmopts

```diff
@@ -1,14 +1,12 @@
-# Cats jvmopts
-# see https://weblogs.java.net/blog/kcpeppe/archive/2013/12/11/case-study-jvm-hotspot-flags
+# Cats jvmopts see https://weblogs.java.net/blog/kcpeppe/archive/2013/12/11/case-study-jvm-hotspot-flags
 -Dfile.encoding=UTF8
--Xms1G
-#-Xmx6G
+-Xms2G
 -Xmx2G
--XX:MaxPermSize=512M
--XX:ReservedCodeCacheSize=250M
+-XX:MaxMetaspaceSize=512M
+-XX:ReservedCodeCacheSize=128M
```

Comment:
The CodeCacheSize parameter has to do with the JIT Code Cache. Since most of our code (tests) is only runned once, we can use a smaller one.